package exceptionHandling;

public class TestClass {

	public static void main(String[] args) 
	{
		//throw new ArithmeticException();
		
		/*int a=0;
		int b=20;
		
		System.out.println(b/a);

		System.out.println("Anand");*/
	}
	
	public static void fun1()
	{
		throw new ArithmeticException();
	}

	public static void fun2()
	{
		throw new IOException();
	}
}
